shell/content/docs/en-us/whats-new.md

```diff
@@ -13,9 +13,9 @@ installed by administrators. Extensions are provided as Helm charts. A new top-l
 
 - Authentication for OCI-based registries is now supported. Note that the structure of the fleet.yaml is the same and the credentials are provided as a Kubernetes secret, which is described in the Private Helm Repo box in the Repo Structure docs.
 
-### k3s provisioning is GA
+### k3s provisioning is GA (x86 Only)
 
-- You can now provision clusters that use k3s. 
+- Provisioning K3s clusters on x86 clusters has graduated to GA! The fully compliant Kubernetes distribution is simplified, secure and at less than 60mb is perfect for Edge.
 
 ---
 title: What's New in 2.6
```

Comment:
When are we branching for 2.7 and 2.6 in UI? I only want "what's new in  2.7" in 2.7 and "what's new in 2.6" in 2.6, I wouldn't want it in both.

Reply:
I'll remove the 2.6.x bits from here for 2.7, though it may have to linger in 2.6.x (somehow).

Reply:
Need to cherry pick this PR once done into `release-2.7.0`. Need to think about what to do for 2.6.x releases.

Reply:
FWIW 

`master` --> go forward branch (post 2.7.0)
`release-2.7.0` --> 2.7.0 branch
`release-2.6.9` --> 2.6.9 branch (which will be the base for anything post 2.6.9)

The `release-2.6.9` branch won't contain any details of 2.7.0